doc/api/container_api.rst

```diff
@@ -0,0 +1,11 @@
+*********
+container
+*********
+
+:mod:`matplotlib.container`
+====================
```

Comment:
Looks like the docs aren't building because this underline needs to be the same length as the above line.

Happy for this to be merged when the travis doc build is passing.